db/migrate/20150928132655_create_monthly_report_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateMonthlyReportComments < ActiveRecord::Migration
+  def change
+    create_table :monthly_report_comments do |t|
+      t.belongs_to :user, null: false
+      t.text :comment
+      t.belongs_to :monthly_report, null: false
+
+      t.timestamps null: false
```

Comment:
__[reek]__
* [FeatureEnvy] refers to t more than self (maybe move it to another class?)